corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yml

```diff
@@ -150,25 +157,30 @@
 - id: use_grid_menus
   name: Modules Menu Display
   description: Display root menu as a list or grid. Read more on the <a target="_blank" href="https://help.commcarehq.org/display/commcarepublic/Grid+View+for+Form+and+Module+Screens">Help Site</a>.
-  values: [false, true]
-  value_names: ['List', 'Grid']
+  values:
+    - ['false', 'List']
+    - ['true', 'Grid']
```

Comment:
@kaapstorm I'm seeing an error on staging that looks like it was caused by this change.  Guessing that needs to be `false` and `true`, the boolean literals, not strings.  This fails when making a change to the "advanced settings" tab of an app.  See details here: https://dimagi-dev.atlassian.net/browse/QA-4462
Example error: https://sentry.io/organizations/dimagi/issues/3522875969/?environment=staging&project=136860&query=is%3Aunresolved

Reply:
Thank you! Let me make sure I didn't make the same mistake anywhere else, and I'll open a PR to fix this.

Reply:
Fixed in https://github.com/dimagi/commcare-hq/pull/32048